src/bin/check.ts

```diff
@@ -28,20 +25,21 @@ export async function check(options: Options) {
     process.exit(1);
   }
   const config = loadBaseConfig(options);
-  for (const filePath of options._) {
-    if (!fs.existsSync(filePath)) {
+  const checkFile = async (filePath: string) => {
```

Comment:
the function is only for this context, move out will make it away from it's used place. And the function is not reusable for other place.

Reply:
The function does 2 things:
* Validating file path;
* Branching file path: `prcessFile` or `processDirectory`

So, I think it can be named `processPathname`.